packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -175,11 +179,44 @@ export interface ChatMessageProps
 
   /** Positions an actionMenu slot in "fixed" mode. */
   unstable_overflow?: boolean;
+
+  /**
+   * Properties that are only available to the v2 (refresh) version of
+   * ChatMessage because they only have an effect there. Enabling this property
+   * will opt into the new layout.
+   */
+  v2?: ChatMessageV2Props;
+}
+
+interface ChatMessageV2Props {
+  /** Indicates whether the message is in a failure state. */
+  failed?: boolean;
+
+  /** Label that describes message importance. */
+  importantLabel?: React.ReactNode;
+
+  /** Additional label slot for message importance, scheduled messages, etc. */
+  infoLabel?: React.ReactNode;
```

Comment:
What's the difference between this and the importance label?

Reply:
Both can coexist at the same time and serve different purposes. I'm not happy with the naming, and would really prefer to _not_ introduce this slot at all. Let's talk about this today.